pkg/virtctl/create/clone/clone_test.go

```diff
@@ -0,0 +1,198 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package clone_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/create/clone"
+	"kubevirt.io/kubevirt/tests/clientcmd"
+)
+
+const (
+	create             = "create"
+	sourceVM, targetVM = "sourceVM", "targetVM"
+)
+
+var _ = Describe("create clone", func() {
+	Context("required arguments", func() {
+
+		const (
+			withSourceFlag, withoutSourceFlag = true, false
+			withTargetFlag, withoutTargetFlag = true, false
+		)
+
+		DescribeTable("with flags defined:", func(addSourceFlag, addTargetFlag bool) {
+			var flags []string
+
+			if addSourceFlag {
+				flags = addFlag(flags, clone.SourceFlag, sourceVM)
+			}
+			if addTargetFlag {
+				flags = addFlag(flags, clone.TargetFlag, targetVM)
+			}
+
+			expectErr := !addSourceFlag || !addTargetFlag
```

Comment:
nit: I'd simply move this condition in the if statement below for readability

Reply:
done :)
thanks!